pkg/provider/okta/okta.go

```diff
@@ -557,10 +557,13 @@ func (oc *Client) follow(ctx context.Context, req *http.Request, loginDetails *c
 		logger.WithField("type", "saml-response").Debug("doc detect")
 		handler = oc.handleFormRedirect
 	} else {
-		req, err = http.NewRequest("GET", loginDetails.URL, nil)
+		req.URL, err = url.Parse(loginDetails.URL)
```

Comment:
is this the way to keep the previous cookie?

Reply:
That's the only thing I came up with :)
I'm very much a Go newbie, so there may be a better way